src/base/Cellar.sol

```diff
@@ -76,12 +78,9 @@ contract Cellar is ERC4626, Ownable, Multicall {
     /**
      * @notice Data related to a position.
      * @param positionType value specifying the interface a position uses
-     * @param highWatermark amount representing the balance this position needs to exceed during the
-     *                      next accrual to receive performance fees
      */
     struct PositionData {
```

Comment:
Since this struct now only stores a single value, delete this struct and change `getPositionData` to `mapping(address => PositionType) getPositionType`.